### DISCUSSION TITLE
Filter PRs created by bots / group similar PRs

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#5491] [Pull Requests]
Filter PRs created by bots like renovate or dependabot.
I often see repositories with more than 10 open PRs created by bots and they are all very similar and not much informative.
To reduce the noise, those PRs can be grouped and / or there can be a filter to hide bot PRs.

**Comment by jhutchings1** [comment]
We don't support this yet, but the request to add grouped updates in Dependabot is tracked by https://github.com/dependabot/dependabot-core/issues/1190 in the Dependabot Core repo.

**Comment by marvi** [comment]
Filter events by user would be a good feature for Notifications. It's not only Dependabot who causes storms of updates. 
It would make notifications much more usable.

**Comment by thesuperzapper** [comment]
@Zerotask @jhutchings1 @marvi I have raised a request on the github product feedback page:

- https://github.com/orgs/community/discussions/122852

Please upvote it if you want to see an `author_is_bot:<bool>` filter for pull requests.
